content/post/why-should-we-trust-ml.md

```diff
@@ -0,0 +1,74 @@
+
+## How can we bridge the gap?
+
+### LIME
+
+LIME approximates a black-box model with a local linear model to obtain an *individual/personalized prediction*. An explanation of an individual prediction is a local linear approximation of the models' behavior in the vicinity of that instance. 
+
+![](/img/lime.png)
+
+How does LIME work? Once your classifier is fit and you identify an instance that you want to understand, LIME samples additional data points in the neighborhood around your instance, uses the pre-fit classifier to identify which classes those samples are in, and builds a logistic regression model on all the features to develop a linear approximation to the classification around that data point. The resulting coefficients of the logistic regression add interpretation around why your data point is getting its particular prediction. LIME is completely model agnostic, and has been used with everything from image classification, text classification, and deep learning prediction.
+
+An open-source package to implement LIME can be found here: https://github.com/marcotcr/lime
+
+### Use Case: Clinical Operations
+
+Consider the use case in which we want to predict which hospitals will enroll more patients. We have many features, and find that what drives hospital enrollment is not very simple, and involves many interactions between variables, as each hospital is different. We build a black-box model that far out performs any linear model in terms of predictive accuracy. Our client is very satisfied with our predictive performance, but wants to understand how we can interpret the predictions of this model, without sacrificing any predictive power. Specifically, what drives the **direction** of the predicted enrollment rate? How can we get personalized explanations? Specifically, why Hospital B gets more enrollment than Hospital A?
+
+By applying LIME, we approximate a linear model for both the predictions of Hospital A and B. We recieve coefficeint estimates for our features around the local regions of these predictions.
+
+![](/img/lime_clinical.png)
+
+We can see that Hospital B has a higher predicted enrollment rate by our black box model and our approximate localized linear model. But why? With LIME, we can see that feature G is what contributes to the difference in enrollment rates. An analysis of that feature and how it differs between the two hospitals would help us understand why these enrollment rates are different, while also giving credit to the other positive features, and understanding that the number of trials in a city actually has a strong negative effect across both hospitals.
```

Comment:
I think "two patients" might be a more clear example, and would be in-line with your introduction in the `## Why is Interpretability Important?` section